### PR TITLE
docker-monolithic: copy from cardano-explorer (deprecated)

### PR DESCRIPTION
**Pull request by disassembler**
This adds the monolithic docker container some exchanges are currently using in a deprecated state (to be replaced with the multi-container approach at some point).

# Issue Number

# Overview

- [ ] I have ...

# Comments

### CODE REVIEW

**Comment by KtorZ** [comment]
@disassembler What's the status on this one?

File: nix/sources.json [hunk 1]
```diff
@@ -1,14 +1,26 @@
...
+        "repo": "cardano-db-sync",
+        "rev": "be2fb8ae73c3ad872de0ad80621bca4718b64b25",
+        "sha256": "1yvy4628bn8a12cx6id4rq2srwgf18z39pbvhg71kfng5wbpx9ys",
+        "type": "tarball",
+        "url": "https://github.com/input-output-hk/cardano-db-sync/archive/be2fb8ae73c3ad872de0ad80621bca4718b64b25.tar.gz",
+        "url_template": "https://github.com/<owner>/<repo>/archive/<rev>.tar.gz"
+    },
     "cardano-node": {
-        "branch": "master",
+        "branch": "tags/1.9.3",
```
Comment (johnalotoski) [inline nix/sources.json:15]: Is this cardano-node tag still the desired target?

File: nix/docker-monolithic/default.nix [hunk 2]
```diff
@@ -0,0 +1,464 @@
...
+          usage = "COUNTER";
+          description = "total blocks in table";
+        };
+      }];
+    };
+  };
+
+  configuration = { config, ... }: {
+    imports = [
+      (cardano-node-src + "/nix/nixos")
```
Comment (johnalotoski) [inline nix/docker-monolithic/default.nix:53]: Duplicate line

File: nix/docker-monolithic/default.nix [hunk 3]
```diff
@@ -0,0 +1,464 @@
...
+      socketPath = "/run/cardano-node/node.socket";
+    };
+
+    services.postgresql = {
+      enable = true;
+      authentication = lib.mkBefore ''
+        local all postgres trust
+      '';
+        enableTCPIP = false;
+        extraConfig = ''
```
Comment (johnalotoski) [inline nix/docker-monolithic/default.nix:93]: This postgres config implies a requirement of >= 8 GB system RAM.  Could consider adding a note for that requirement to the appropriate README.

**Comment by rhyslbw** [comment]
PR is stale, will retain branch

**Pull request closed without merging**